kube-enforcer/templates/mutating-webhook.yaml

```diff
@@ -33,5 +33,15 @@ webhooks:
         path: "/mutate"
     timeoutSeconds: {{ .Values.webhooks.mutatingWebhook.timeout }}
     admissionReviewVersions: ["v1beta1"]
+    sideEffects: "None"
+    {{- if eq .Values.global.platform "openshift"}}
```

Comment:
If I understand correctly, if the conditional block {{- if eq .Values.global.platform "openshift"}} evaluates to false, the namespaceSelector block will not be included in the rendered output.

As the namespaceSelector is not exclusive to OpenShift, the conditional block {{- if eq .Values.global.platform "openshift"}} is unnecessary.